### ISSUE TITLE
URLs in NPM page are broken

### ISSUE THREAD

**Issue by ajsaraujo** [maximelafarie/ngx-smart-modal#311]
In https://www.npmjs.com/package/ngx-smart-modal, the `Documentation` and `Demo` hyperlinks point to https://maximelafarie/ngx-smart-modal and https://maximelafarie/ngx-smart-modal/#/demo, respectively, which aren't valid urls.

**Comment by vibonacci** [comment]
.com missing

**Comment by ajsaraujo** [comment]
Exactly. That's how the URLs are set in the NPM page.

**Comment by khylias** [comment]
Indeed, the links of the demonstration and documentation on NPM are not up to date. A commit was made after the last release of the package on NPM. 

@maximelafarie Is an update scheduled soon?

**Comment by maximelafarie** [comment]
Yup @khylias, I'm preparing a big refactor in several steps and this one is included in.
